### ISSUE TITLE
output of generate_tsv is not accepted by Y.Toloka

### ISSUE THREAD

**Issue by AleksandrShepelev** [innosoft-pro/label-them#16]
tested on branch develop-toloka.
ya-disk:
![image](https://cloud.githubusercontent.com/assets/14818993/26714510/1eca0de4-478b-11e7-8329-605abc5aadf3.png)

proof that picture is opened:
![image](https://cloud.githubusercontent.com/assets/14818993/26714527/3eb130b0-478b-11e7-9f67-a3d4e97699e0.png)

tsv-file:
```
INPUT:image_rel	INPUT:json_params	GOLDEN:result
/LT-Data1/20161106_230949_7777_00408CACB891.mkv_004.jpg		true
/LT-Data1/20161106_230949_7777_00408CACB891.mkv_005.jpg		true
/LT-Data1/20161106_230949_7777_00408CACB891.mkv_003.jpg		true
/LT-Data1/20161106_230949_7777_00408CACB891.mkv_006.jpg		true
```

toloka errors:
![image](https://cloud.githubusercontent.com/assets/14818993/26714598/9e643f5c-478b-11e7-84b1-0ad982a16c90.png)

```
{
    "withFailure": true,
    "results": {
        "2": {
            "success": false,
            "payload": {
                "tasks.0.input_values.json_params": {
                    "code": "VALUE_REQUIRED",
                    "message": "Value must be present and not equal to null"
                }
            }
        },
        "3": {
            "success": false,
            "payload": {
                "tasks.0.input_values.json_params": {
                    "code": "VALUE_REQUIRED",
                    "message": "Value must be present and not equal to null"
                }
            }
        },
        "4": {
            "success": false,
            "payload": {
                "tasks.0.input_values.json_params": {
                    "code": "VALUE_REQUIRED",
                    "message": "Value must be present and not equal to null"
                }
            }
        },
        "5": {
            "success": false,
            "payload": {
                "tasks.0.input_values.json_params": {
                    "code": "VALUE_REQUIRED",
                    "message": "Value must be present and not equal to null"
                }
            }
        }
    }
}
```

**Comment by ruslanmustafin** [comment]
Hi, it looks like your TSV file is missing JSON params. Did you specify a JSON file with no contents?

**Comment by ruslanmustafin** [comment]
Ok, the problem was in the script, it is fixed now (see 90dbaf0ce4dbe31b44c74757e334d6c7c9102a95)

**Comment by nmingazov** [comment]
Worked on my machine. Should be ok now.
